### PR TITLE
Features/hdev 593 add subgraph support to other chain

### PR DESCRIPTION
**Pull request by geonellov**
## Description of Changes
Support matic and arbitrum for TVL ticker

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-593)

### Developer Checklist:

* [X] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [x] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Comment by schystz** [comment]
@geonellov initial feedback upon testing the FE:

- for arb & matic, let's show mainnet TVL on Vesting page with the mainnet indicator we had before, just so we don't display 0
- can we fix the conflict on `src/halo-hooks/useTVLInfo.ts`?
- on arb Farm page, TVL is 19.2K while the liquidity on the table shows 19.3K - maybe its just rounding difference, let's see if we can fix it and make it the same

File: src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx [hunk 1]
```diff
@@ -66,11 +63,6 @@ const EpochReleaseTimerCard = ({ event, content, countdown }: TimeObject) => {
             <p className="text-12px bold ">DEPOSIT IN POOL </p>
             <h4 className="text-lg">{content.deposit}</h4>
           </div>
-          {chainId !== ChainId.MAINNET && (
```
Comment (schystz) [inline src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx]: Let's preserve this on networks != mainnet, as mentioned on the prior comment

**Review by schystz — COMMENTED** [review]
Looks good @geonellov ! Nice quick work!

File: src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx (hunk 1 above)
Comment (geonellov) [reply src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx]: @schystz done

**Comment by geonellov** [comment]
> @geonellov initial feedback upon testing the FE:
> 
> * for arb & matic, let's show mainnet TVL on Vesting page with the mainnet indicator we had before, just so we don't display 0
> * can we fix the conflict on `src/halo-hooks/useTVLInfo.ts`?
> * on arb Farm page, TVL is 19.2K while the liquidity on the table shows 19.3K - maybe its just rounding difference, let's see if we can fix it and make it the same

@schystz done

**Review by schystz — APPROVED** [review]

**Review by tracyarciaga — APPROVED** [review]

**Pull request merged**